### ISSUE TITLE
Chord in gp5 file not shown

### ISSUE THREAD

**Issue by jakobabesser** [Perlence/PyGuitarPro#6]
Hi, thanks for your efforts on the pyguitarpro library.
I tried to import a simple bass line from a GP5 file (exported from GP6) with one chord on the first beat. However I find the `note.beat.effect.chord` to be `None` for all notes.
Github doesn't allow to attach gp5 files so I'm happy to send it to you.
How can I parse the chord label on beat 1?
Thanks for your help!

**Comment by Perlence** [comment]
Hey, you are welcome! Upload the tab to some file sharing host like https://transfer.sh/ and I'll give it a look.

**Comment by jakobabesser** [comment]
You find the file in https://www.dropbox.com/s/4zviwgxch52vhld/001_Funky_Guy.gp5?dl=0 
It's the Gm7 on beat 1. I found a similar behavior for other files exported from Guitar Pro 6 in the gp5 format.

**Comment by Perlence** [comment]
@jakobabesser I fixed the problem in 41aaee2. But take another look at the chord, it doesn't have fingering. Probably GP6->GP5 conversion is messed up.

**Comment by jakobabesser** [comment]
Works like a charm, thanks! When writing the guitar pro file in GuitarPro 6, I unselected the "show fingering" checkbox in the chord diagram editor. This might explain the missing fingering in the file.
I agree, the gp5 export is buggy. Sidenote: I also noticed that left hand fingerings are not exported from GP 6 to the gp5 nor MusicXML files. I reported this bug to the official site, let's see what happens.
